migrations/Version202109081901591888_taoMediaManager.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\model\Middleware\Context\MiddlewareContext;
+use oat\tao\model\Middleware\Context\OpenApiMiddlewareContext;
+use oat\tao\model\Middleware\MiddlewareManager;
+use oat\tao\model\Middleware\MiddlewareRequestHandler;
+use oat\tao\model\Middleware\OpenAPISchemaValidateRequestMiddleware;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202109081901591888_taoMediaManager extends AbstractMigration
+{
+    private const OPENAPI_SPEC_PATH = ROOT_PATH . '/taoMediaManager/doc/taoMediaManagerApi.yml';
+
+    public function getDescription(): string
+    {
+        return 'OpenAPI Schema validation middleware registration for SharedStimulus API';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $openApiMiddleware = $this->getServiceManager()->get(OpenAPISchemaValidateRequestMiddleware::SERVICE_ID)
+            ->addSchema($this->getOpenApiMiddlewareContext());
+
+        $middlewareHandler = $this->getServiceManager()->get(MiddlewareManager::class)
+            ->append($this->getMiddlewareContext())->getMiddlewareHandler();
+
+        $this->getServiceManager()->register(OpenAPISchemaValidateRequestMiddleware::SERVICE_ID, $openApiMiddleware);
+        $this->getServiceManager()->register(MiddlewareRequestHandler::SERVICE_ID, $middlewareHandler);
+    }
```

Comment:
This extension should only contain this:

```suggestion
    public function up(Schema $schema): void
    {
        $openApiMiddleware = $this->getServiceManager()->get(OpenAPISchemaValidateRequestMiddleware::SERVICE_ID)
            ->addSchema($this->getOpenApiMiddlewareContext());

        $this->getServiceManager()->register(OpenAPISchemaValidateRequestMiddleware::SERVICE_ID, $openApiMiddleware);
    }
```